### PR TITLE
Configuration update - Nanometrics Centaur and Kinemetrics EpiSensor FBA-ES-T configuration

### PR DESCRIPTION
**Pull request by CBurton90**

### CODE REVIEW

**Review by salichon — COMMENTED** [review]
looks legit,
Response needs to be checked independently i reckon :)

**Comment by salichon** [comment]
This will need  a final  `auto.go` file creation with this combo  

this is set there:  https://github.com/GeoNet/delta/pull/1137

**Review by salichon — APPROVED** [review]

**Comment by salichon** [comment]
@ozym  
I assign myself since @CBurton90 can not merge.
(so - i ll wait for your review(s) ( to go on)
thanks cheers

File: resp/responses/configurations/strongmotion.yaml [hunk 1]
```diff
@@ -316,6 +316,27 @@ response:
...
+  Nanometrics Dataloggers Connected to Strong Motion Sensors:
+    sensors:
+    - sensors:
+      - FBA-ES-T
+      - FBA-ES-T-ISO
+      filters:
+      - FBA-ES-T
+      channels: ZNE
+      reversed: false
+    - dataloggers:
```
Comment (ozym) [inline resp/responses/configurations/strongmotion.yaml:328]: `dataloggers:` is missing

**Review by ozym — CHANGES_REQUESTED** [review]

File: resp/responses/configurations/strongmotion.yaml (hunk 1 above)
Comment (CBurton90) [reply resp/responses/configurations/strongmotion.yaml:329]: ah yes good spot, I've added it now @ozym

**Review by salichon — APPROVED** [review]

**Comment by salichon** [comment]
follow up  (auto.go) is in  https://github.com/GeoNet/delta/pull/1137

**Review by ozym — APPROVED** [review]

**Pull request merged**